src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Model.Route.yml

```diff
@@ -0,0 +1,21 @@
+Symfony\Cmf\Bundle\RoutingBundle\Model\Route:
```

Comment:
same here, I think we should create our class which extends `Symfony\Cmf\Bundle\RoutingBundle\Model\Route` and import our directories in jms serializer